backend/src/test/java/com/example/parkingspot/controller/PersonControllerMVCTest.java

```diff
@@ -0,0 +1,137 @@
+package com.example.parkingspot.controller;
+
+import java.util.List;
+
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import org.mockito.ArgumentMatchers;
+import org.mockito.Mockito;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.autoconfigure.web.servlet.AutoConfigureMockMvc;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import org.springframework.http.MediaType;
+import org.springframework.test.web.servlet.MockMvc;
+import org.springframework.test.web.servlet.RequestBuilder;
+import org.springframework.test.web.servlet.request.MockMvcRequestBuilders;
+import org.springframework.test.web.servlet.result.MockMvcResultMatchers;
+
+import com.example.parkingspot.entity.Car;
+import com.example.parkingspot.entity.Person;
+import com.example.parkingspot.service.CarService;
+import com.example.parkingspot.service.PersonService;
+
+@WebMvcTest(PersonController.class)
+@AutoConfigureMockMvc(addFilters = false)
```

Comment:
Nice to have: somehow get user mock to work to test spring security implementation.